app/subscriber/src/features/search-page/components/advanced-search/components/sections/MediaSection.tsx

```diff
@@ -1,13 +1,16 @@
 import { useFilterOptions } from 'components/navbar/hooks';
 import { IoIosArrowDropdownCircle, IoIosArrowDroprightCircle } from 'react-icons/io';
 import { useContent } from 'store/hooks';
-import { Col, Row, Select, Show } from 'tno-core';
+import { Button, ButtonVariant, Checkbox, Col, Row, Select, Show } from 'tno-core';
 
 import { SubMediaGroups } from '../../constants';
 import { ISubMediaGroupExpanded } from '../../interfaces';
+import { IFilterDisplayProps } from './IFilterDisplayProps';
 import { determineSelectedMedia, sortableMediaOptions } from './utils';
 
-export interface IMediaSectionProps {
+export interface IMediaSectionProps extends IFilterDisplayProps {
+  // /** whether to display filter choices as checkboxes or a select box */
+  // displayFiltersAsDropdown: boolean;
```

Comment:
Something doesn't look right here.  Below it's not complaining this is commented out.

Reply:
good catch, I was experimenting with extending the interface.